### ISSUE TITLE
Evaporation output 

### ISSUE THREAD

**Issue by Bkemanga** [aemon-j/LakeEnsemblR#282]
I was wondering if it would be possible to calibrate the GLM model lakeEnsemblr using observed evaporation.

**Comment by jorritmesman** [comment]
Hi,
No, it's sadly not possible to run an automatic calibration with anything else than temperature in LakeEnsemblR, at least for now. I would recommend to set up a latin hypercube grid with the parameters that you want to calibrate, run GLM iteratively over these parameters, and each step calculate your goodness-of-fit metric(s), so that you can select the best fit in the end. You can partially follow the cali_ensemble.R script, perhaps, but it will be mostly something that you have to create yourself, I'm afraid.

The key function to do this, also used in `cali_ensemble`, is the `Latinhyper` function from the `FME` package.

**Comment by Bkemanga** [comment]
Thanks very much @jorritmesman  
One quick question: is it possible to run Ensemble with GLM only for 1000? also, I have only temperature observation for the lake surface (depth =0m, that is, surface water temperature), Is it possible to calibrate the model using Cali_ensemble?

**Comment by Bkemanga** [comment]
I will appreciate your guidance as am trying to replicate this for mine pit lake.

**Comment by jorritmesman** [comment]
You are able to run more iterations; there is no maximum limit for the `num` argument (other than runtime on your computer). 

Yes, you should be able to use cali_ensemble to calibrate your lake. However, having only surface water temperature is of course a bit restrictive to get stratification patterns correct, because you have no data on that. But at least theoretically, it should be possible.

**Comment by jorritmesman** [comment]
Yes, that will work fine. You don't need to specify anything in the `cali_ensemble` function call, but just make sure that the observation file that you refer to only has observations in the period that you want to use for calibration.

**Comment by Bkemanga** [comment]
I did manage to set my lakemodel but when i rune the model I get this error 
> # Run ensemble lake models
> run_ensemble("NPMEnsemblR.yaml", model = c("GLM"))
Loading water temperature observations...[2024-05-02 04:04:54.365859]
Finished loading water temperature observations![2024-05-02 04:04:54.367365]
Running models... (Have you tried parallelizing?) [2024-05-02 04:04:54.378466]
GLM run is complete! [2024-05-02 04:04:54.402798]
**Error in R_nc4_open: No such file or directory
Error in nc_open(file, readunlim = TRUE) : 
  Error in nc_open trying to open file ./GLM/output/output.nc (return_on_error= FALSE )**

**Comment by jorritmesman** [comment]
It looks like GLM crashed (no output file was generated). You can investigate what went wrong by going to your GLM folder and using `GLM3r::run_glm(verbose = TRUE)`

**Comment by Bkemanga** [comment]
@jorritmesman I did manged to figure out the error after using GLM3r::run_glm(verbose = TRUE) as you did advice. 
Now another problem, when using .yaml file to export the the setting using export_config, the exported meteo data does generate longwave radiation data but I would  preferer to use Cloud cover direct when run GLM so that i can specify different cloud_mode. In short i don't want the export_config to calculate longwave.
